tests/rpc_test.go

```diff
@@ -0,0 +1,37 @@
+package tests
```

Comment:
I thought we were going to mirror the package structure in the tests directory?

Reply:
Yes sorry, Fixed it now